### PR TITLE
Add Student model to students app

### PR DESCRIPTION
**Pull request by ValarieChebet**

### CODE REVIEW

File: yearbook/yearbook/settings.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+]
```
Comment (Okemwag) [inline yearbook/yearbook/settings.py:41]: You forgot to register your students app in the INSTALLED_APP list

File: yearbook/students/models.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+class Student(models.Model):
+    name = models.CharField(max_length=100)
+    student_class = models.CharField(max_length=50)
+    email = models.EmailField(unique=True)
+    photo = models.ImageField(upload_to='photos/')
+
+    def __str__(self):
+        return self.name
+
```
Comment (Okemwag) [inline yearbook/students/models.py:12]: Consider adding a Class Meta to list order of student based on class

**Review by Okemwag — CHANGES_REQUESTED** [review]

**Comment by Okemwag** [comment]
Kindly make changes as per the recommendations then we can merge
@ValarieChebet

**Review by Okemwag — APPROVED** [review]

File: yearbook/students/models.py (hunk 2 above)
Comment (ValarieChebet) [reply yearbook/students/models.py:12]: Thank you

**Pull request merged**